contributors/devel/staging.md

```diff
@@ -0,0 +1,34 @@
+# Staging Directory and Publishing
+
+The [staging/ directory](https://github.com/kubernetes/kubernetes/tree/master/staging) of Kubernetes contains a number of pseudo repositories ("staging repos"). They are symlinked into Kubernetes' [vendor/ directory](https://github.com/kubernetes/kubernetes/tree/master/vendor/k8s.io) for Golang to pick them up.
```

Comment:
Let's use git.k8s.io urls:
- https://git.k8s.io/kubernetes/staging
- https://git.k8s.io/kubernetes/vendor/k8s.io

Reply:
Why don't we have these redirects without the `git.` prefix? Would make go package map to github pages.